lib/cater/service.rb

```diff
@@ -66,10 +72,20 @@ def after_error(*filters, &blk)
         set_callback(:error, :after, *filters, &blk)
       end
 
+      def required(&block)
+        self.input_validators.send(:required, &block)
+      end
+
+      def input_validators
```

Comment:
@andriytyurnikov this is here, because I want required block to be defined before call method, not inside. I was failed while trying to move it somewhere else and still be usable. If you have another ideas for this, I will be happy to discuss them